app/main.py

```diff
@@ -1 +1,60 @@
 #  write your code here
+class Animal:
+
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry is True and self.appetite != 0:
```

Comment:
I think this condition can be simplified, for example like this: `if self.is_hungry:`, because if this condition is True, it will be fulfilled